jwql/tests/test_dark_monitor.py

```diff
@@ -122,6 +128,8 @@ def test_noise_check():
     assert np.all(noisy[1] == np.array([3, 9]))
 
 
+@pytest.mark.skipif(os.path.expanduser('~') == '/home/jenkins',
```

Comment:
Similar for this test. It seems like it doesn't require any information that isn't available when running on Jenkins.